index.html

```diff
@@ -42,7 +42,159 @@ <h1>I'm Denzel Glad to see you!</h1>
+                        <img src="/images/Tag.png" alt="css-projects">
+                        <img src="/images/Tag-2.png" alt="javaScritp-projects">
+
+                    </div>
+
+                    <div class="external_links">
+                        <button type="button" class="view_project">
+                            See Projects
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project">
+                <div class="image_div">
+                    <img src="/images/Snapshoot-Portfolio-1.png" alt="project image" class="project_image">
+                </div>
+                <div class="details_div">
+                    <h2 class="project_title">Multi-Post
+                        Stories</h2>
+                    <ul class="some_details">
+                        <li class="bold">CANPOY</li>
+                        <li class="tech"><i class="fas fa-circle"></i>Back End Dev</li>
+                        <li class="tech"><i class="fas fa-circle"></i>2015</li>
+                    </ul>
+                    <p class="project_details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="technologies_used">
+
+                        <img src="/images/Tag-1.png" alt="html-projects">
+                        <img src="/images/Tag.png" alt="css-projects">
+                        <img src="/images/Tag-2.png" alt="javaScritp-projects">
+
+                    </div>
+
+                    <div class="external_links">
+                        <button type="button" class="view_project">
+                            See Projects
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div class="project">
+                <div class="image_div">
+                    <img src="/images/3.png" alt="project image" class="project_image">
+                </div>
+                <div class="details_div">
+                    <h2 class="project_title">Tonic</h2>
+                    <ul class="some_details">
+                        <li class="bold">CANPOY</li>
+                        <li class="tech"><i class="fas fa-circle"></i>Back End Dev</li>
+                        <li class="tech"><i class="fas fa-circle"></i>2015</li>
+                    </ul>
+                    <p class="project_details">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                    <div class="technologies_used">
+
+                        <img src="/images/Tag-1.png" alt="html-projects">
+                        <img src="/images/Tag.png" alt="css-projects">
+                        <img src="/images/Tag-2.png" alt="javaScritp-projects">
+
+                    </div>
```

Comment:
The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`.
Instead of using `div` and `img` tags, you should use `ul` and `li`. Fix the same for the rest of your list of project technologies.

- [ ] Fix all your list of project technologies using the tags required for this project.